JekyllBlogCommentsAzure/WebConfigurator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Configuration;
+
+namespace JekyllBlogCommentsAzure
+{
+    public class WebConfigurator : IConfiguration
+    {
+        public string CommentWebsiteUrl => ConfigurationManager.AppSettings["CommentWebsiteUrl"];
+
+        public string GitHubToken => ConfigurationManager.AppSettings["GitHubToken"];
+
+        public string PullRequestRepository => ConfigurationManager.AppSettings["PullRequestRepository"];
+
+        public string CommentFallbackCommitEmail => ConfigurationManager.AppSettings["CommentFallbackCommitEmail"];
+
+        public string SentimentAnalysisSubscriptionKey => ConfigurationManager.AppSettings["SentimentAnalysis.SubscriptionKey"];
+
+        public string SentimentAnalysisRegion => ConfigurationManager.AppSettings["SentimentAnalysis.Region"];
+
+        public string SentimentAnalysisLang => ConfigurationManager.AppSettings["SentimentAlaysis.Lang"];
```

Comment:
Heh heh, a typo here.

```suggestion
        public string SentimentAnalysisLang => ConfigurationManager.AppSettings["SentimentAnalysis.Lang"];
```

Reply:
Ohhhhhh I didn't saw this...
Thanks to review this @Haacked